layouts/partials/comment.html

```diff
@@ -95,10 +95,11 @@
             {{- if $telegram.outlined -}}
                 {{- $attr = printf `%v data-outlined="1"` $attr -}}
             {{- end -}}
-            {{- dict "Source" "https://comments.app/js/widget.js?2" "Defer" true "Attr" $attr | dict "Scratch" .Scratch "Data" | partial "scratch/script.html" -}}
+            <script type="text/javascript" src="https://comments.app/js/widget.js?3" {{ with $attr }} {{ . | safeHTMLAttr }}{{ end }}></script>
```

Comment:
I think you don't need to remove `defer` attribute. It make main page page load earlier as the script is executed when the page has finished parsing.

Reply:
> I think you don't need to remove `defer` attribute. It make main page page load earlier as the script is executed when the page has finished parsing.

Thank you very much for this fix, I see you marked in my PR, but this is not your template, the developer has abandoned this project?